src/components/Search.js

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components/macro'
+import Button from './Button'
+
+//require('dotenv').config()
+const API_KEY = process.env.REACT_APP_API_KEY
+
+export default function Search({ onSubmit }) {
+  const [options, setOptions] = useState([])
+  const [query, setQuery] = useState('')
+  const [selectedOption, setSelectedOption] = useState(null)
+  useEffect(() => {
+    setSelectedOption(null)
+    fetch(`https://api.rawg.io/api/games?key=${API_KEY}&search=${query}`)
+      .then(res => res.json())
+      .then(data => setOptions(data.results))
+  }, [query])
+  return (
+    <section>
+      <Form onSubmit={handleSubmit}>
+        <label>
+          Selected Game: {selectedOption?.name || 'None'}
+          <input
+            autoComplete="off"
+            name="gamename"
+            id="auto"
+            placeholder="Type to search"
+            value={selectedOption?.name}
+            onChange={event => setQuery(event.target.value)}
+          />
+        </label>
+        <Button>Add game</Button>
+      </Form>
+      <Suggestions className="autoContainer">
+        {query &&
+          options.map(option => {
+            return (
+              <div
+                onClick={() => setSelectedOption(option)}
+                className="option"
+                key={option.id}
+              >
+                <span>{option.name}</span>
+              </div>
+            )
+          })}
+      </Suggestions>
+    </section>
```

Comment:
If you don't style the `section` you could just use fragments.
```suggestion
    <>
      <Form onSubmit={handleSubmit}>
        <label>
          Selected Game: {selectedOption?.name || 'None'}
          <input
            autoComplete="off"
            name="gamename"
            id="auto"
            placeholder="Type to search"
            value={selectedOption?.name}
            onChange={event => setQuery(event.target.value)}
          />
        </label>
        <Button>Add game</Button>
      </Form>
      <Suggestions className="autoContainer">
        {query &&
          options.map(option => {
            return (
              <div
                onClick={() => setSelectedOption(option)}
                className="option"
                key={option.id}
              >
                <span>{option.name}</span>
              </div>
            )
          })}
      </Suggestions>
    </>
```